### PR TITLE
Clipboard Editor extension

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

Allow the user to edit it's clipboard content using a form.

## Screencast

clipboard-editor

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

It works as expected, could you look into removing the drafts marks on the logo and then we're good to go 🔥

**Comment by pomdtr** [comment]
Thanks for the review @pernielsentikaer

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Comment by sming** [comment]
👋 @pomdtr , pernielsentikaer ,

TL;DR I’d like to extend/enhance this extension to add a command e.g. `Edit Clipboard & Search` which, perhaps unsurprisingly, performs a web search immediately after the user completes editing the clipboard contents. Why? Because I do this **all the time**, no joke.

Would extending this extension be the right approach? Or should I write my own that leverages this one? I haven’t written a public Raycast extension before and don’t know what the protocols are in this repo by and large.

Cheers,
Pete
